run/test/setup.ts

```diff
@@ -152,12 +141,12 @@ describe('setup', function() {
       other: Stream<number>;
     };
 
-    function app(sources: TestSources) {
+    function app(so: TestSources) {
```

Comment:
Why this change?

Reply:
This was probably testing frustration, I can revert.